assets/scss/templates/_header.scss

```diff
@@ -0,0 +1,398 @@
+/* header */
+$dp-header-height: 4em;
+$dp-border-color: #CCCCCC;
+$dp-border-radius: 5px;
+$dp-text-aling: center; /* header */
+
+.header-main {
+  height: $dp-header-height;
+  border-bottom: 1px solid $dp-border-color;
+  box-shadow: 0 0 0 4px $dp-color-snow;
+  background: #FFF;
+  position: fixed;
+  width: 100%;
+  left: 0;
+  top: 0;
+  z-index: 100;
+  display: flex;
```

Comment:
Please use Atomic classes for this cases, maybe, make a meeting with Fede to set which ones to create.